extensions/management/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -25,9 +25,8 @@ name: "Camel Management"
 description: "JMX management strategy and associated managed resources"
 icon-url: "https://camel.apache.org/_/img/logo-d-f21b25ba38.svg"
 metadata:
-  unlisted: true
```

Comment:
Is JMX supported on GraalVM now? I know it was planned, I just want to make sure it happened already.

Reply:
Yes it is supported starting from v 23 but it is experimental

Reply:
It has been added by this PR https://github.com/oracle/graal/issues/4732